brightway/export_agb/export.py

```diff
@@ -160,6 +162,8 @@ def build_product_tree(ciqual_products, max_products=None):
                 )
                 products[product_name][step][exchange_category] = category_data
 
+                # If we are at the "at plant" step and there is a transport process we
```

Comment:
Is this comment related to the one just below? If so remove the empty line, and dedent one level to match the indentation level of line 167.
If not, maybe it's missing the end of the sentence?

Reply:
oups forgot to delete this comment - fixed now